buildSrc/src/main/kotlin/com/google/devtools/ksp/ApiCheck.kt

```diff
@@ -31,6 +32,16 @@ fun Project.configureMetalava() {
         task.description = "Check API compatibility."
         task.group = "Verification"
         task.args = listOf("--check-compatibility:api:released", API_BASE_FILE) + task.args!!
+        task.inputs.files(API_BASE_FILE).withPropertyName("apiCheckBaseFile").withPathSensitivity(PathSensitivity.RELATIVE)
```

Comment:
Can you remind me again if we decided that we had properly accounted for all possible inputs and outputs of this task?

Reply:
We did, yes.